staging/src/k8s.io/apiserver/plugin/pkg/authenticator/token/oidc/oidc.go

```diff
@@ -302,35 +260,30 @@ func New(opts Options) (*Authenticator, error) {
 	}
 
 	verifierConfig := &oidc.Config{
-		ClientID:             opts.ClientID,
+		ClientID:             opts.JWTAuthenticator.Issuer.ClientIDs[0],
```

Comment:
Validate that the length of this slice is equal to `1`.

Reply:
Done in validation!